src/app.module.ts

```diff
@@ -13,9 +11,7 @@ import { MongooseModule } from "@nestjs/mongoose";
     ConfigModule.forRoot({ isGlobal: true }),
     HealthModule,
     CatsModule,
-    MongooseModule.forRoot("mongodb://localhost/test"),
+    MongooseModule.forRoot(process.env.CATS_URL),
```

Comment:
Is it the URL only for the demo cats api? Why it is named CATS_URL?

Reply:
the URL is changed to DATABASE_URL